modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -689,6 +850,59 @@ private ServiceContext _createServiceContext(
 		return serviceContext;
 	}
 
+	private Map<String, ObjectRelationship> _getAvailableObjectRelationshipMap(
+		ObjectDefinition objectDefinition) {
+
+		Map<String, ObjectRelationship> objectRelationshipMap = new HashMap<>();
+
+		List<ObjectRelationship> availableObjectRelationships =
+			_objectRelationshipLocalService.getObjectRelationships(
```

Comment:
I think it would be better to request with the possible relationship name. For that, I see there is a method in the ObjectRelationshipLocalServiceImpl `getObjectRelationshipByObjectDefinitionId`, but that would throws an exception if the relationship does not exist.

My proposal is to rename the `fetchObjectRelationshipByObjectDefinitionId` of that class to `fetchObjectRelationshipByObjectDefinitionId1` (as it uses the `objectDefinition1` field) and then, create the `fetchObjectRelationshipByObjectDefinitionId` with the similar functionality of `getObjectRelationshipByObjectDefinitionId` but without throwing the exception

Once we have that method, we would get the relationships whose name matches with the rest of the fields that have not been managed by the `_toObjectValues` method.

We could discuss it in person better, I think it will be easier to understand